nuxeo-vision-aws/src/main/resources/OSGI-INF/aws-rekognition-provider.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.vision.aws.provider" version="1.0">
+
+  <require>org.nuxeo.vision.core.service</require>
```

Comment:
Same as Google Vision provider; no need to require a component for contributing to it